Development environment setup/Local Database.md

```diff
@@ -0,0 +1,7 @@
+As Laravel Herd doesn't come with the Database setup we recommend using Dbngin. It's a simple and lightweight database server for macOS. You can download it from [here](https://dbngin.com/).
+
+After installation you only need to start __mysql__ service.
+
+By default all your databases will be available at `localhost:3306`.
+
+For previewing the databases we recommend using [TablePlus](https://tableplus.com/) or any other database management tool.
```

Comment:
```suggestion
For previewing the databases we recommend using [Sequel Ace](https://sequel-ace.com/), [TablePlus](https://tableplus.com/) or any other database management tool.
```
Sequel Ace is open-source, completely free and feature packed so I'd add it as a first choice 😄

Reply:
Just use `mysql-cli` like a true hackerman 😄

Reply:
Or something like this if you are using Neovim 😎
https://github.com/kndndrj/nvim-dbee